packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableClaimableRow.tsx

```diff
@@ -326,35 +318,16 @@ export function TransactionsTableClaimableRow({
     () =>
       sanitizeTokenSymbol(tx.asset, {
         erc20L1Address: tx.tokenAddress,
-        chain: getWagmiChain(
-          isSourceChainIdEthereum ? tx.parentChainId : tx.childChainId
-        )
```

Comment:
this condition is important for USDC.e vs USDC so it shouldn't be removed

<img width="900" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/77b52edd-40e4-4f19-b542-50aa9ca35b6d">

vs 

<img width="1005" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/cb4b63ef-07de-4531-8664-50271f4f9566">

Reply:
btw the row bg colour doesn't have a clear pattern to the users now, should check with designer

Reply:
Yeah let's confirm tomorrow and address in a different PR

Reply:
Reverted the logic for USDC.e